lib/beaker/hypervisor/openstack.rb

```diff
@@ -220,16 +220,15 @@ def provision
 
       @hosts.each do |host|
         ip = get_ip
-        host[:vmhostname] = ip.ip.gsub '.','-'
-        host[:vmfqdn] = host[:vmhostname] + '.rfc1918.puppetlabs.net'
```

Comment:
@kevpl backwards compatible in the sense that the output was the same a Lucy's original. By all means kill the fqdn! When I run it my adresses are not RFC1918 and definitely not part of puppetlabs.net :-)